src/js/services/dialer.js

```diff
@@ -1,7 +1,10 @@
+import Backbone from 'backbone';
 import Radio from 'backbone.radio';
 
 import App from 'js/base/app';
 
+const patients = new Backbone.Collection([]);
```

Comment:
Collection that the buttons are based on.

`{ id: String, name: String },`

Seems to automatically show/hide buttons when patients are added and removed. And when the entire collection is emptied.